crates/macro-support/src/parser.rs

```diff
@@ -1568,10 +1577,22 @@ impl MacroParse<&ClassMarker> for &mut syn::ImplItemFn {
             let kind = operation_kind(&opts);
             ast::MethodKind::Operation(ast::Operation { is_static, kind })
         };
+
+        // Validate that js_namespace is not used on methods
```

Comment:
How do I tell that we are working on methods here? Is it:
```
impl MacroParse<&ClassMarker> for &mut syn::ImplItemFn {
```
and `syn::ImplItemFn` is a `fn` node?

Reply:
Yes, because `impl` blocks are specifically for struct functions which are by definition methods or static methods.